openedx/adg/lms/webinars/tests/test_tasks.py

```diff
@@ -0,0 +1,28 @@
+"""
+All tests for webinar tasks
+"""
+from datetime import datetime, timedelta
+from unittest.mock import patch
+
+import pytest
+
+from ..tasks import task_reschedule_webinar_reminders
+from .factories import WebinarFactory, WebinarRegistrationFactory
+
+
+@pytest.mark.django_db
+@patch('openedx.adg.lms.webinars.tasks.reschedule_webinar_reminders')
+@pytest.mark.parametrize(
+    'new_start_time, expected_call_count', [
+        (datetime.now() + timedelta(days=1), 1),
+        (datetime.now() + timedelta(days=8), 2),
+    ]
+)
+def test_task_send_mandrill_email(mock_reschedule, new_start_time, expected_call_count):
```

Comment:
Lets rename it all params have the same outcome For example `test_task_send_mandrill_email_successfully`

If you both parametrize have different outcome then let's define it in params